src/tests/test_dynamic_api/test_serializers.py

```diff
@@ -625,7 +626,54 @@ def test_indirect_self_reference(ligplaatsen_model, filled_router):
         serializer_factory(ligplaatsen_model)
 
     @staticmethod
-    def test_field_permissions_display_first_letter(
+    def test_auth_link_item(drf_request, afval_container_model, afval_container):
+        """Prove that the ``_links`` field also handles permissions."""
+        ContainerSerializer = serializer_factory(afval_container_model)
+        container_serializer = ContainerSerializer(
+            afval_container, context={"request": drf_request}
+        )
+        data = normalize_data(container_serializer.data)
+        assert data == {
+            "_links": {
+                # The cluster block is completely omitted because there is no access to the table.
+                # So won't expose the identifier either. This isn't ambiguous either,
+                # because if the relation was missing, it would be "cluster": null,
```

Comment:
Reconstruction of a comment that I just posted, but that GitHub seems to have swallowed:

```suggestion
                # Just showing the href would leak the identifier. Setting "cluster": null
                # would be ambiguous, since that already means "no cluster for this container".
```

Reply:
thanks! I was struggling with that last commit all the time.